conflowgen/analyses/yard_capacity_analysis.py

```diff
@@ -54,7 +50,9 @@ def get_used_yard_capacity_over_time(
             smoothen_peaks: Whether to smoothen the peaks.
             convert_to_teu: Whether to convert the used yard capacity to TEU.
         Returns:
-            A series of the used yard capacity in TEU over the time.
+            YardCapacity: A namedtuple consisting of two dictionaries. The first dictionary represents the
```

Comment:
I would like to keep the term `YardCapacity` for some kind of future feature and not already use it here. What we talk about here is the used yard capacity over time which is a series of data, not a single datum. A bit more verbose but probably worth it would be `UsedYardCapacityOverTime` - then it would also neatly match with the method name! What do you think?

Reply:
Yup, that makes sense.